examples/go/gateway-docs/go.mod

```
@@ -0,0 +1,39 @@
+module gateway-docs
```

Please rename the example package to something meaningful. 
```suggestion
module github.com/vendasta/api-gateway-docs/tools/gateway-docs
```